.github/workflows/cypress.yml

```
@@ -35,28 +35,45 @@ jobs:
         working-directory: ${{ env.KEYCLOAK_SERVER_PATH }}
         run: cp .github/settings.xml ~/.m2
 
-      - name: Build Admin UI
-        working-directory: ${{ env.ADMIN_UI_PATH }}
-        run: mvn clean install --batch-mode --file keycloak-theme/pom.xml
+      - name: Build theme
+        working-directory: ${{ env.ADMIN_UI_PATH }}/keycloak-theme
+        run: mvn clean install --batch-mode
+
+      - name: Determine theme path
+        working-directory: ${{ env.ADMIN_UI_PATH }}/keycloak-theme
+        run: |
+          VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+          echo "THEME_JAR_PATH=admin-ui/keycloak-theme/target/keycloak-admin-ui-$VERSION.jar" >> $GITHUB_ENV
 
```

This isn't needed I'd say
```suggestion
      - name: Determine theme path
        working-directory: ${{ env.ADMIN_UI_PATH }}/keycloak-theme
        run: |
          VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
          echo "THEME_JAR_PATH=admin-ui/keycloak-theme/target/keycloak-admin-ui-$VERSION.jar" >> $GITHUB_ENV

```